Features/foutafhandeling.feature

```diff
@@ -190,6 +198,89 @@ Abstract Scenario: Ongeldige queryparameter waarde bij zoeken
     En is in het antwoord status=406
     En is in het antwoord code=notAcceptable
 
+  Scenario: Invalide request body
+    Gegeven op endpoint /panden kan met POST gezocht worden op <locatie>
+    Als een request body wordt aangeboden die niet vertaald kan worden naar een valide locatie
+    Dan is de http status code van het antwoord 422
+    En is in het antwoord title=Request body bevat een ongeldige waarde.
+    En is in het antwoord status=422
+    En is in het antwoord code=unprocessableEntity
+
+  Scenario: Technische- of interne fout
+    Gegeven de API heeft last van een technisch probleem bij een verzoek van de gebruiker
+    Als een endpoint wordt aangesproken
+    Dan is de http status code van het antwoord 500
+    En is in het antwoord title=Interne server fout.
+    En is in het antwoord status=500
+    En is in het antwoord code=serverError
+
+  Scenario: Bronservice is niet beschikbaar
+    Als een endpoint wordt geraadpleegd
+    En de BAG API geen response of een timeout geeft
+    Dan is de http status code van het antwoord 503
+    En is in het antwoord title=Bronservice is niet beschikbaar.
+    En is in het antwoord status=503
+
+  Scenario: CRS in bbox-crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'bbox-crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter bbox-crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+    
+  Scenario: CRS in filter-crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'filter-crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter filter-crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+
+  Scenario: CRS in crs parameter niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in parameter crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=paramCrsNotSupported
+
+  Scenario: CRS in Content-Crs header wordt niet ondersteund
+    Als een pand wordt geraadpleegd met parameter 'Content-Crs'=b
+    Dan is de http status code in het antwoord 400
+    En is in het antwoord title=CRS b in header Content-Crs wordt niet ondersteund.
+    En is in het antwoord status=400
+    En is in het antwoord code=headerCrsNotSupported
```

Comment:
Dit scenario zou een breaking change op de huidige functionaliteit zijn (http 415, code crsNotSupported).
Wat we wel kunnen doen is de POST geometrie endpoints deprecaten en dit scenario in een volgende major versie van de api implementeren.